app.js

```diff
@@ -9,4 +9,40 @@ burger.addEventListener('click', () => {
 document.querySelectorAll('.nav-links').forEach((n) => n.addEventListener('click', () => {
   burger.classList.remove('active');
   navList.classList.remove('active');
-}));
\ No newline at end of file
+}));
+
+const form = document.getElementById('form');
+
+const email = document.getElementById('email');
+
+// function setErrorFor(message) {
+//   const formControl = document.querySelector('error');
+//   const small = formControl.querySelector('small');
+
+//   // add error message inside small
+//   small.innerText = message;
+//   // add error class
+//   // formControl.classList = 'error';
+// }
```

Comment:
- kindly Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.👌👍